### DISCUSSION TITLE
Logging cost is too expensive.

### DISCUSSION THREAD

**Discussion by meotimdihia** [vercel/vercel#6033] [Ideas] [answered]
Hi, I want to say about Log Drain.
Currently, services that can be integrated with Vercel are too expensive. It can excess my hosting cost, just for logging.

I think about 2 solutions: 
1. If I could filter requests with a header with an HTTP code is 200, then I can decrease logging cost. I just need to log the error request
2. Or I could Integrate with New Relic, Google Cloud Logging, Grafana because their price is affordable - the free tier is very good too.

**Comment by leerob** [comment, marked as answer]
Have you seen the latest integrations for logging? There are many with a very generous free tier. I have personally used Logflare and Logtail and they are very nice!

https://vercel.com/integrations#logging
